peachjam/templates/peachjam/_quick_search.html

```diff
@@ -1,9 +1,10 @@
 {% load i18n %}
 <form action="{% url 'search:search' %}" method="get" class="d-flex mb-1">
   <input type="text"
+         aria-label="{% trans 'Search' %}{{ court.name }} {{ author.name }} {{ nature.name }} {{ registry.name }} {% if MULTIPLE_JURISDICTIONS and country %}{{ country.name }}{% endif %} {{ locality.name }} {{ year }}"
          class="form-control flex-grow-1"
          name="q"
-         placeholder="Search {{ court.name }} {{ author.name }} {{ nature.name }} {{ registry.name }} {% if MULTIPLE_JURISDICTIONS and country %}{{ country.name }}{% endif %} {{ locality.name }} {{ year }} "/>
+         placeholder="{% trans 'Search ' %}{{ court.name }} {{ author.name }} {{ nature.name }} {{ registry.name }} {% if MULTIPLE_JURISDICTIONS and country %}{{ country.name }}{% endif %} {{ locality.name }} {{ year }} "/>
```

Comment:
Don't have the space in the trans, it'll get lost.

Reply:
noted, fixed